src/interface/src/app/plan/create-scenarios/create-scenarios.component.spec.ts

```diff
@@ -87,7 +90,10 @@ describe('CreateScenariosComponent', () => {
             questions: [
               {
                 short_question_text: 'test_question',
-                priorities: ['test_priority'],
+                scenario_output_fields: [''],
+                scenario_priorities: ['test_priority'],
```

Comment:
"priorities" in ScenarioConfig is replace with "scenario_priorities" (to make it easier for George to parse on the backend)

Reply:
sorry I meant that we are populating this with `test_priority`- is that correct? If its temp can we add a todo or a note on why?

Reply:
Ohh sorry I saw this comment on mobile and thought you were commenting on a line above. I just had a filler string in there for the test — I'll change it to a blank string to avoid confusion.